src/modules/common/components/Page.js

```diff
@@ -112,7 +112,7 @@ class Page extends React.Component<PropType, StateType> {
     const {title, children, content, files, theme, language, cityCode, lastUpdate} = this.props
     const height = this.state.webViewHeight
     const width = this.state.webViewWidth
-
+    const dataDetectorTypes: any = 'all' // flow doesn't recognize this as type of DataDetectorTypes
```

Comment:
Use a FlowFixMe instead of a workaround with any.

Reply:
Flow is super dumb in this case as well, btw. We should think about upgrading it soonish... I hope they improved their stuff a little bit at least :D